spec/schemas/instance.yaml

```
@@ -26,18 +26,30 @@ InstanceSpec:
   - bootVolume
   properties:
     skuRef:
-      $ref: './resource.yaml#/Reference'
+      allOf:
+      - $ref: './resource.yaml#/Reference'
+      - description: Reference to the SKU of the instance.
+      - example: 'skus/seca.s'
     primaryNicRef:
-      $ref: './resource.yaml#/Reference'
+      allOf:
+      - $ref: './resource.yaml#/Reference'
+      - description: Reference to the primary NIC attached to this instance.
+      - example: 'network-interfaces/nic-123'
```

the short form is only allowed within a provider, but nic, security-group and storage are other providers, so you need to change the examples to include the provider, e.g.:
```suggestion
      allOf:
      - $ref: './resource.yaml#/Reference'
      - description: Reference to the primary NIC attached to this instance.
        example: 
          provider: "seca.network/v1"
          resource: 'network-interfaces/nic-123'
```

Please adopt the other parts in this file as well.